components/concept/engine/src/main/java/mozilla/components/concept/engine/manifest/WebAppManifest.kt

```diff
@@ -57,7 +57,7 @@ data class WebAppManifest(
     val dir: TextDirection = TextDirection.AUTO,
     val lang: String? = null,
     val orientation: Orientation = Orientation.ANY,
-    val scope: String? = null,
+    val scope: String = startUrl,
```

Comment:
If we always get one then should we just not have any default at all?

Reply:
I left a default to make writing tests a bit easier and avoid breaking changes. We could break and make this a required parameter instead.